plugins/github-actions/src/components/WorkflowRunDetailsPage/useWorkflowRunJobs.ts

```diff
@@ -13,5 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { useAsync } from 'react-use';
+import { Jobs } from '../types';
 
-export { BuildInfoCard } from './BuildInfoCard';
+export const useWorkflowRunJobs = (jobsUrl?: string) => {
+  const jobs = useAsync<Jobs>(async () => {
+    if (jobsUrl === undefined) return [];
+    const data = await fetch(jobsUrl).then(d => d.json());
```

Comment:
Wait, shouldn't it be part of the GithubActionsApi?

Reply:
Probably. I think I just went with the octokit flow. I'll give it another look.

Reply:
I think it's fine to keep this logic outside of Api but Jobs type should be part of it.